modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/layout/index.html

```diff
@@ -1,61 +1,110 @@
 <div class="bg-brand-primary-lighten-6 d-flex layout-container">
-<div class="bg-brand-primary-darken-5 d-lg-flex d-none flex-column justify-content-between ray-sidebar-container ray-sticky-top sticky-top">
+	<div
+		class="bg-brand-primary-darken-5 d-lg-flex d-none flex-column justify-content-between ray-sidebar-container ray-sticky-top sticky-top"
```

Comment:
Do you really need "ray-sticky-top sticky-top"?

Reply:
Yes.